app/models/ability.rb

```diff
@@ -9,7 +9,7 @@ def initialize(user)
     admin_user_permissions(user) if user.role_admin?
     programm_user_permissions(user) if user.role_programm?
 
-    if user.pbs_id
+    if user.pbs_id && !user.pbs_id.zero?
```

Comment:
this is needed since there are users that have `pbs_id = 0`, could be cleaned up